src/identity-providers/routes/IdentityProvider.ts

```diff
@@ -1,43 +1,26 @@
 import type { LocationDescriptorObject } from "history";
 import { generatePath } from "react-router-dom";
 import type { RouteDef } from "../../route-config";
-import { AddIdentityProvider } from "../add/AddIdentityProvider";
 import { DetailSettings } from "../add/DetailSettings";
 
 type IdentityProviderTabs = "settings" | "mappers";
```

Comment:
Like in other places in the name of a type is generally singular.
```suggestion
type IdentityProviderTab = "settings" | "mappers";
```